package-system/vulkan-validationlayers/Findvulkan-validationlayers.cmake

```diff
@@ -0,0 +1,40 @@
+#
+# All or portions of this file Copyright (c) Amazon.com, Inc. or its affiliates or
+# its licensors.
+#
+# For complete copyright and license terms please see the LICENSE at the root of this
+# distribution (the "License"). All use of this software is governed by the License,
+# or, if provided, by the license below or the license accompanying this file. Do not
+# remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+
+set(MY_NAME "vulkan-validationlayers")
+
+set(TARGET_WITH_NAMESPACE "3rdParty::${MY_NAME}")
+if (TARGET ${TARGET_WITH_NAMESPACE})
+    return()
+endif()
+
+add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
+
+if (${PAL_PLATFORM_NAME} STREQUAL "Windows")
+    set(PATH_TO_DLL ${CMAKE_CURRENT_LIST_DIR}/vulkan-validationlayers/lib/release)
+    
+    set(${MY_NAME}_RELEASE_RUNTIME_DEPENDENCIES
+        ${PATH_TO_DLL}/VkLayer_khronos_validation.dll
+    )
+
+    set(${MY_NAME}_RUNTIME_JSON_DEPENDENCIES
+        ${PATH_TO_DLL}/VkLayer_khronos_validation.json
+    )
+
+    ly_add_target_files(
+        TARGETS 
+        ${TARGET_WITH_NAMESPACE} 
+        FILES 
+        ${${MY_NAME}_RELEASE_RUNTIME_DEPENDENCIES} 
+        ${${MY_NAME}_RUNTIME_JSON_DEPENDENCIES})
+endif()
```

Comment:
This would add the dependencies for copies, I think, but in CMAKE terms the corect way to do it is to set
IMPORTED_LOCATION property on the SHARED IMPORTED GLOBAL library.
in otehr words
```
set(_DLL_NAME ${PATH_TO_DLL}/VkLayer_khronos_validation${CMAKE_SHARED_LIBRARY_SUFFIX})
# ${CMAKE_SHARED_LIBRARY_SUFFIX} is '.dll' on windows but appropriately named on other systems.

# Declare that its a SHARED library (ie, a dll) as opposed to a STATIC (lib) or INTERFACE (header only)
# declaring it as IMPORTED means we are going to ingest a pre-existing thing, rather than build one.
add_library(${TARGET_WITH_NAMESPACE} SHARED IMPORTED GLOBAL)

# declare that the primary imported location (ie, the dll for a shared, the lib for a static) is the dll
# the imported location is basically "What to import?".  For a shared library, its the dll.
set_target_properties(${TARGET_WITH_NAMESPACE} PROPERTIES IMPORTED_LOCATION ${_DLL_NAME})

# files that cmake doesn't care about / are not involved in the build system have to be handled custom.
ly_add_target_files(
        TARGETS 
        ${TARGET_WITH_NAMESPACE} 
        FILES 
         ${${MY_NAME}_RUNTIME_JSON_DEPENDENCIES})
```